### ISSUE TITLE
Roadmap?

### ISSUE THREAD

**Issue by alexbw** [andresy/torch9#2]
Is this the official torch9 repo? If so, what's the roadmap for release and expected improvements over torch7?

**Comment by hughperkins** [comment]
Yes, what is torch9? :-O  And shouldnt there be like a torch8? :-P

**Comment by FuriouslyCurious** [comment]
Is Torch9 development abandoned now? Thanks for the update!

**Comment by metab0t** [comment]
Any update on this? 
Is the development abandoned at present?
